<div class="row" style="margin-left: 15px">
    <h4 class="center" style="text-align: center"><span style="color: #FDB933;font-weight: bold">TF databases</span></h4>
    <hr class="divider">
    <table class="two" cellpadding="6">
        <tr><td style="width: 15%">
            <a href="hTFtarget/" target=_blank><Img class="thumbnail" src="/static/lab_new/graph/htftarget_resource.jpg" width="300" height="200" style='border-style:none;margin-right: 5px;margin-top: 5px'/></a></td><td style="width: 35%">
            <p style="font-family: Arial; text-align:justify; text-justify:distribute-all-lines;">Identifying the target genes of TFs is very important in the study of gene expression regulation, development and disease pathogenesis. In this hTFtarget database, we collected comprehensive human TF ChIP-Seq data and customized an analysis workflow to identify reliable TF targets with taking epigenomic states into account.
            </p></td>
            <td style="width: 15%">
                <a href="AnimalTFDB" target=_blank><Img class="thumbnail" src="/static/lab_new/graph/animaltfdb_resource.jpg" width="300" height="200" style='border-style:none;margin-right: 5px;'/></a></td><td style="width: 35%">
                <p style="font-family: Arial; text-align:justify; text-justify:distribute-all-lines;">AnimalTFDB is a comprehensive database including classification and annotation of genome-wide transcription factors (TFs), transcription co-factors and chromatin remodeling factors in 97 animal genomes. The TFs are further classified into 73 families based on their DNA-binding domain (DBD).The 97 animal species were classified into 11 categories according to the Ensembl taxonomy. The former version of AnimalTFDB database is<a href="AnimalTFDB2/index.php"> here</a>
                </p></td></tr>
    </table>
    <h4 class="center" style="text-align: center"><span style="color: #FDB933;font-weight: bold">ncRNA databases</span></h4>
    <hr class="divider">
    <table class="two" cellpadding="6">
        <tr><td style="width: 15%">
            <a href="lncRNASNP2/" target=_blank><Img class="thumbnail" src="/static/lab_new/graph/lncrnasnp_resource.jpg" width="300" height="200" style='border-style:none;margin-right: 5px'/></a></td><td style="width: 35%">
            <p style="font-family: Arial; text-align:justify; text-justify:distribute-all-lines;width:100%">LncRNASNP is a database to provide a comprehensive resource of single nucleotide polymorphisms (SNPs) in human lincRNAs. It contains SNPs in lncRNAs, structure change by SNP, the binding sites of lncRNA:miRNA and potential functional SNPs which can impact lncRNA:miRNA interaction.Furthermore, it provides mutations and their effects in lncRNAs, including mutations from COSMIC and TCGA. The first version of lncRNASNP database is <a href="lncRNASNP/" target="_blank">here</a></p>
        </td>
            <td style="width: 15%">
                <a href="LNCediting/" target=_blank><Img class="thumbnail" src="/static/lab_new/graph/lncediting_resource.jpg" width="300" height="200" style='border-style:none;margin-right: 5px'/></a></td><td style="width: 35%">
                <p style="font-family: Arial; text-align:justify; text-justify:distribute-all-lines;width:100%">LNCediting is a database to provide a comprehensive resource for the functional prediction of RNA editing in long non-coding RNAs (lncRNAs) across four species, including human, mouse, rhesus and fly. It contains editing sites in lncRNAs, structure change by editing and the binding sites of lncRNA:miRNA which may be impacted by editing sites. In total, we identified 195,907 RNA editing sites in four lncRNA transcripts and found thousands of editing sites which greatly impact the secondary structure of lncRNA and the binding of miRNA-lncRNA.</p></td></tr>
        <tr><td style="width: 15%">
            <a href="lncRInter/" target=_blank><Img class="thumbnail" src="/static/lab_new/graph/lncrinter_resource.jpg" width="300" height="200" style='border-style:none;margin-right: 5px'/></a></td><td style="width: 35%">
            <p style="font-family: Arial; text-align:justify; text-justify:distribute-all-lines;">lncRInter is reliable and high quality lncRNA interaction database containing experimentally validated data. The lncRNA interaction datasets are all extractd from peer-reviewed publications by our curators with strict criteria that there were certain experimental evidence (e.g. RNA pull-down, luciferase reporter assay, in vitro binding assay, etc.) for direct lncRNA interactions.</p></td>

            <td style="width: 15%">
                <a href="SNORic/" target=_blank><Img class="thumbnail" src="/static/lab_new/graph/snoric_resource.jpg" width="300" height="200" style='border-style:none;margin-right: 5px'/></a></td><td style="width: 35%">
                <p style="font-family: Arial; text-align:justify; text-justify:distribute-all-lines;">The SNORic refers to "snoRNA in cancers". Small nucleolar RNAs (snoRNAs) are a class of regulatory non-coding RNAs, which play a conserved role in ribosome biogenesis. However, in recent years, increasing evidence has demonstrated that snoRNAs play important roles in tumorigenesis. Here, we systematically analyzed snoRNA expression profiles in > 10,000 samples across 31 cancer types from The Cancer Genome Atlas, and analyzed their correlation with other TCGA genomic, proteomic, epigenomic and clinical data (e.g. patient tumor stage, subtype, and survival).</p></td></tr>
    </table>
    <h4 class="center" style="text-align: center"><span style="color: #FDB933;font-weight: bold">miRNA databases</span></h4>
    <hr class="divider">
    <table class="two" cellpadding="6">
        <tr><td style="width: 15%">
            <a href="miRNASNP2/index.php" target=_blank><Img class="thumbnail" src="/static/lab_new/graph/mirnasnp_resource.jpg" width="300" height="200" style='border-style:none;margin-right: 5px;margin-top: 5px'/></a></td><td style="width: 35%"> <p style="font-family: Arial; text-align:justify; text-justify:distribute-all-lines;">miRNASNP: The aim of the microRNA-related SNP Database (miRNASNP) is to provide the lastest resource of the miRNA-related SNPs, which included SNPs in human pre-miRNAs and miRNA flanks, SNPs in other species's miRNAs, target gain and loss by SNPs in miRNA seed regions or 3'UTR of target mRNAs. Thus, Five major modules are provided in this database and users can browse or search it in different levels.
        </p></td>
            <td style="width: 15%">
                <a href="smallRNA/index.php" target=_blank><Img class="thumbnail" src="/static/lab_new/graph/hmed_resource.jpg" width="300" height="200" style='border-style:none;margin-right: 5px;margin-top: 5px'/></a></td><td style="width: 35%"><p style="font-family: Arial; text-align:justify; text-justify:distribute-all-lines;"> Human MicroRNA Expression Database (HMED) is an integrative database providing expression information of human miRNAs. HMED displays the miRNA expression in more than 400 data sets from smRNA data in NCBI GEO and SRA, containing samples from various diseases and tissues. The aim of HMED is to provide valuable resources of miRNA for researchers.  </p></td></tr>
        <tr><td style="width: 15%">
            <a href="http://bioinfo.life.hust.edu.cn/miR_path" target=_blank><Img class="thumbnail" src="/static/lab_new/graph/mirpath_resource.jpg" width="300" height="200" style='border-style:none;margin-right: 5px;margin-top: 5px'/></a></td><td style="width: 35%"> <p style="font-family: Arial; text-align:justify; text-justify:distribute-all-lines;"> This is a database with user-friendly web interface to display the miRNA pathway regulation and their expressions in the 20 tumor types. The database provides search, browse and download function for the the miR-pathway data. Users can search miRNA regulating pathways, miRNA target genes and the expression profiles of miRNAs and genes in TCGA.
        </p></td>

            <td style="width: 15%">
                <a href="http://bioinfo.life.hust.edu.cn/EVmiRNA" target=_blank><Img class="thumbnail" src="/static/lab_new/graph/evmir_resource.jpg" width="300" height="200" style='border-style:none;margin-right: 5px;margin-top: 5px'/></a></td><td style="width: 35%"> <p style="font-family: Arial; text-align:justify; text-justify:distribute-all-lines;">Extracellular vesicles (EVs) released by living cells include exosomes and microvesicles (MVs), which contain various molecules from parental cells and are potential sources for disease diagnostic and therapeutic biomarkers. miRNA is the most well-studied molecule type in EVs because of its important functions and biomarker properties. Thus, we build the Extracellular Vesicles miRNA database (EVmiRNA) to collect comprehensive miRNA expression profiles in EVs.
            </p></td></tr>
    </table>
    <h4 class="center" style="text-align: center"><span style="color: #FDB933;font-weight: bold">eQTL databases</span></h4>
    <hr class="divider">
    <table class="two" style="width:50%"  cellpadding="6">
        <tr><td style="width: 3%">
            <a href="PancanQTL/" target=_blank><Img class="thumbnail" src="/static/lab_new/graph/pancanqtl_resource.jpg" width="300" height="200" style='border-style:none;margin-right: 5px'/></a></td><td style="width: 47%">
            <p style="font-family: Arial; text-align:justify; text-justify:distribute-all-lines;">Expression quantitative trait loci (eQTLs) are regions of the genome containing DNA sequence variants that influence the expression level of one or more genes. PancanQTL aims to comprehensively provide cis-eQTLs (SNPs affect local gene expression) and trans-eQTLs (SNPs affect distant gene expression) in 33 cancer types from The Cancer Genome Atlas (TCGA).</p></td></tr>
    </table>
    <h4 class="center" style="text-align: center"><span style="color: #FDB933;font-weight: bold">SEGs and regulation databases</span></h4>
    <hr class="divider">
    <table class="two" cellpadding="6" style="width:50%">
        <tr><td style="width: 3%">
            <a href="SEGreg" target=_blank><Img class="thumbnail" src="/static/lab_new/graph/segreg_resource.jpg" width="300" height="200" style='border-style:none;margin-right: 5px;'/></a></td><td style="width: 47%">
            <p style="font-family: Arial; text-align:justify; text-justify:distribute-all-lines;">SEGreg is a comprehensive database containing SEGs identified by our SEGtool from four high-throughput datasets of different cancers/tissues/cell lines (TCGA, BodyMap, CCLE and GTEx), along with their corresponding miRNA and TF regulations collected from several databases. SEGreg will provide useful clues to reveal the functions and regulations of SEGs.</p>
        </td></tr>
    </table>
    <h4 class="center" style="text-align: center"><span style="color: #FDB933;font-weight: bold">Tools</span></h4>
    <hr class="divider">
    <table class="two" cellpadding="6">
         <tr>
            <td style="width: 15%"><a href="GSCALite" target=_blank><Img class="thumbnail" src="/static/lab_new/graph/gscalite.jpg" width="300" height="200" style='border-style:none;margin-right: 5px;margin-top: 5px'/></a></td><td style="width: 35%"><p style="font-family: Arial; text-align:justify; text-justify:distribute-all-lines;">Gene Set Cancer Analysis (GSCALite) was designed for users to analyze a set of genes in cancers with the following functional modules. (i) Differential expression in tumor versus normal, and the survival analysis; (ii) Genomic variations and their survival analysis; (iii) Gene expression associated cancer pathway activity; (iv) miRNA regulatory network for genes; (v) Drug sensitivity for genes; (vi) Normal tissue expression and eQTL for genes. GSCALite is a user-friendly web server for dynamic analysis and visualization of gene set in cancer and drug sensitivity correlation
</p>
        </td>
            <td style="width: 15%"><a href="SEGtool" target=_blank><Img class="thumbnail" src="/static/lab_new/graph/segtool_resource.jpg" width="300" height="200" style='border-style:none;margin-right: 5px;margin-top: 5px'/></a></td><td style="width: 35%"><p style="font-family: Arial; text-align:justify; text-justify:distribute-all-lines;">SEGtool is an R package with self-adaptive function and high accuracy for specifically expressed gene (SEG, also known as tissue specific gene) detection. SEGs are essentially outliers in a given condition (or different treatments, tissues). In order to detect such outliers, SEGtool combines modified fuzzy c-means (FCM), Jaccard index and greedy annealing methods to detect SEGs.</p>
        </td>
        </tr>
            <tr style="width:50%"><td style="width: 15%"><a href="GSDS" target=_blank><Img class="thumbnail" src="/static/lab_new/graph/gsds_resource.jpg" width="300" height="200" style='border-style:none;margin-right: 5px;margin-top: 5px'/></a></td><td style="width: 35%"><p style="font-family: Arial; text-align:justify; text-justify:distribute-all-lines;">Gene Structure Display Server (GSDS) is designed for the visualization of gene features, such as the composition and position of exons, introns, and conserved elements .etc. The input could be sequences, GenBank Accession Number (or GI), or features in BED/GTF/GFF3 formats. After inputting gene features, a high-quality image can be generated. Shape and color for features can be customized by users and further modifying functions on figures are provided.</p>
        </td>
        </tr>
    </table>
</div>
